server.py

```diff
@@ -99,6 +100,12 @@ def main(services, host, port, users):
         auth_svc = AuthService()
         logging.debug('Uploaded files will be put in %s' % cfg['exfil_dir'])
         file_svc = FileSvc([p.name.lower() for p in plugin_modules], cfg['exfil_dir'])
-        agent_svc = AgentService()
+        agent_svc = AgentService(untrusted_timer=cfg['untrusted_timer'])
```

Comment:
I want our first iteration of this to have untrusted_timer required-- so let's swap file simply to this:
```
agent_svc = AgentService(untrusted_timer=cfg['untrusted_timer'])
logging.debug('Agents will be considered untrusted after %s seconds of silence' % cfg['untrusted_timer'])
```